### ISSUE TITLE
[BUG] Phone fields don't have any CSS styling

### ISSUE THREAD

**Issue by anonhostpi** [danielo515/obsidian-modal-form#284]
![image](https://github.com/danielo515/obsidian-modal-form/assets/130698314/e24d7d24-f088-480e-b5ef-bfabb93c3d6a)

Inspector reveals that the input type is "tel":

![image](https://github.com/danielo515/obsidian-modal-form/assets/130698314/de4b48ec-0b48-4c7e-b47b-71b452d9a2ad)

When inspecting app.css, there is no instance of type="tel" like there is for type="number"

Platform: Windows 11
Obsidian: v1.6.3
Modal Forms: v1.47.0

**Comment by danielo515** [comment]
Yes, that is a problem that Obsidian does not  provide any styling for that input. I will add a simple fast one
